testsuite/test-backup.sh

```diff
@@ -6,10 +6,17 @@ run restic backup "${BASE}/fake-data"
 run restic restore "$(basename "$RESTIC_REPOSITORY"/snapshots/*)" "${BASE}/fake-data-restore"
 dirdiff "${BASE}/fake-data" "${BASE}/fake-data-restore/fake-data"
 
-SNAPSHOT=$(run restic list snapshots)
+SNAPSHOT=$(restic list snapshots)
 run restic backup -p "$SNAPSHOT" "${BASE}/fake-data"
 run restic restore "$(basename "$RESTIC_REPOSITORY"/snapshots/*)" "${BASE}/fake-data-restore-incremental"
 dirdiff "${BASE}/fake-data" "${BASE}/fake-data-restore-incremental/fake-data"
 
+# first line contains snapshot id
+# second line contains timestamp of directory creation
+# so compare from line three on
+echo "snapshot id is $SNAPSHOT"
+restic ls "$SNAPSHOT" | tail -n +3 > "${BASE}/test-ls-output"
```

Comment:
delete the "test-ls-output" file after? or just pipe to diff directly instead of generating a temporary file?